src/bootstrap/sanity.rs

```diff
@@ -78,7 +78,13 @@ pub fn check(build: &mut Build) {
         }
         need_cmd("cmake".as_ref());
         if build.config.ninja {
-            need_cmd("ninja".as_ref())
+            // Some Linux distros rename `ninja` to `ninja-build`.
+            // CMake will accept and work with both, so let's just probe for either.
+            if have_cmd("ninja".as_ref()).is_some() || have_cmd("ninja-build".as_ref()).is_some() {
+                // OK
+            } else {
+                need_cmd("ninja".as_ref)
+            }
```

Comment:
Perhaps this could look like:

```rust
if !have_cmd("ninja-build".as_ref()) {
    need_cmd("ninja".as_ref())
}
```

The comment can just indicate that if `ninja-build` is present we won't need `ninja` because CMake detects both.